src/screens/FeedScreen/index.js

```diff
@@ -79,6 +83,12 @@ const FeedScreen = (props) => {
     }
   }, [interactionsComplete]);
 
+  React.useEffect(() => {
+    if (anonChannels.length > 0) {
+      StorageUtils.totalAnonChannels.set(anonChannels.length?.toString() || '0');
+    }
+  }, [anonChannels.length]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a potential issue with the use of optional chaining (`?.`) on `anonChannels.length`. Since `anonChannels` is an array, its `length` property will always be defined (even if it's 0), so there's no need for optional chaining here. Also, you're converting the length to a string and then storing it in local storage. This could lead to potential type confusion later when retrieving this value. It would be better to store it as a number and convert it to a string only when necessary.
```diff
- StorageUtils.totalAnonChannels.set(anonChannels.length?.toString() || '0');
+ StorageUtils.totalAnonChannels.set(anonChannels.length);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->